builtin/logical/pki/ocsp.go

```diff
@@ -26,6 +27,7 @@ import (
 const (
 	ocspReqParam            = "req"
 	ocspResponseContentType = "application/ocsp-response"
+	maximumRequestSize      = 87 * 5 // A normal simple request is 87 bytes, so give us some buffer
```

Comment:
We could probably safely bump this to say, 2048 just to ensure we give ourselves more space if necessary for spec changes (or if we start accepting more req extensions)

Reply:
I don't disagree, though I think @stevendpclark might give us a way to stop people from trying to check a bunch of serial numbers at once, and this won't break anything for a new feature.

Reply:
Sure, we could lower/up it as we go, basically just a safety valve, bumped to 2048 for now.